lib/checkoutUtils.js

```diff
@@ -13,27 +13,15 @@ export const categories = [
 // Creates a dictionary object from each product.
 function createProductData(record) {
   record = record.fields;
-  if (record['Image']) {
-    return {
-      name: record['Name'],
-      id: record.id,
-      category: record['Category'],
-      points: record['Points'],
-      customerCost: record['Customer Cost'],
-      image: record['Image'][0].url,
-      cartCount: 0
-    };
-  } else {
-    return {
-      name: record['Name'],
-      id: record.id,
-      category: record['Category'],
-      points: record['Points'],
-      customerCost: record['Customer Cost'],
-      image: null,
-      cartCount: 0
-    };
-  }
+  return {
+    name: record['Name'],
+    id: record.id,
+    category: record['Category'],
+    points: record['Points'],
+    customerCost: record['Customer Cost'],
+    image: record['Image'] ? record['Image'][0].url : null,
```

Comment:
I modified this in my PR (since i swapped out all the Airtable calls) but doesn't conflict -- will adapt this in my branch (-: